### DISCUSSION TITLE
Unable to access Sponsorship Listing data from GraphQL API

### DISCUSSION THREAD

**Discussion by PraharshitaK-IS** [community/community#63833] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Getting sponsorship listing data form the GraphQL explorer works fine but stops working when I try to run it from postman or from a python script.

Please find below the response I receive.I am able to get the login, which indicates the query formation and bearer token are working fine.

Any help in this regard would be greatly appreciated.

```
{
    "data": {
        "user": {
            "sponsorsListing": null,
            "login": "kovidgoyal"
        }
    }
}
```

**Comment by octokatherine** [comment]
Hi @PraharshitaK-IS! Could you share the query you are executing?

**Comment by PraharshitaK-IS** [comment]
Hi Katherine @octokatherine 
Please find below

`{
  user(login: "kovidgoyal") {
    login
    
    sponsorsListing {
      createdAt
    }
  }
}`

**Comment by octokatherine** [comment, marked as answer]
Are you using a fine-grained access token, or a classic personal access token? I would try again with a classic token if you're not already using one.

**Comment by PraharshitaK-IS** [comment]
I just switched from fine-grained to classic token and it worked!!
Thanks a lot Katherine! <3
